tests/Core/Tokenizers/Tokenizer/RecurseScopeMapSwitchTokenScopeTest.inc

```diff
@@ -1,26 +1,26 @@
 <?php
 
 /* testSwitchNormalSyntax */
-switch ($value) {
+switch ($value) /* testSwitchNormalSyntaxScopeOpener */ {
     case 1:
         echo 'one';
         break;
     default :
         echo 'other';
         break;
+/* testSwitchNormalSyntaxScopeCloser */
 }
 
 // Test for https://github.com/squizlabs/PHP_CodeSniffer/issues/497
 /* testSwitchAlternativeSyntax */
-switch ($value):
-    /* testSwitchAlternativeSyntaxEnsureTestWillNotPickUpWrongColon */
+switch ($value) /* testSwitchAlternativeSyntaxScopeOpener */ :
```

Comment:
This breaks the test.

```php
/* testSwitchAlternativeSyntax */
switch ($value):
    /* testSwitchAlternativeSyntaxEnsureTestWillNotPickUpWrongColon */
    case 1:
```

☝🏻 Here the `/* testSwitchAlternativeSyntax */` marker is used to find the `:`/`T_COLON` scope opener for the alternative syntax.
The `/* testSwitchAlternativeSyntaxEnsureTestWillNotPickUpWrongColon */` marker is used to prevent the test from incorrectly picking up on the colon after `case 1`.

This is to prevent the test passing if a tokenizer issue would have (incorrectly) tokenized the `:` after `switch ($value)` as `T_INLINE_ELSE`.

In the updated version, that protection is removed.

Reply:
I completely missed why `/* testSwitchAlternativeSyntaxEnsureTestWillNotPickUpWrongColon */` was there in the first place. Thanks for the explanation. I have now restored this marker and removed the unnecessary scope opener one.